modules/light-clients/06-solomachine/types/client_state.go

```diff
@@ -39,6 +39,20 @@ func (cs ClientState) GetLatestHeight() exported.Height {
 	return clienttypes.NewHeight(0, cs.Sequence)
 }
 
+// GetTimestampAtHeight returns 0.
```

Comment:
Is this comment correct? The implementation seems to indicate that a non-zero timestamp could also be returned...

Reply:
Ah, forgot to update the comment here... :man_facepalming: